bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -219,6 +251,10 @@ private function createOrderTask(GsOrder $gsOrder)
                 'issuetype' => [
                     'id' => $this->params->get('gs_order_issuetype_id'),
                 ],
+                'reporter' => [
+                    'name' => $author,
+                ],
+                'customfield_10309' => (string) $gsOrder->getDebitor(),
```

Comment:
Custom field ids can change between instances.
See https://github.com/aakb/jira_economics/blob/develop/bundles/Billing/Service/BillingService.php#L806
for an example of how to get the id of the custom field.

Reply:
Added the field id .env instead. I believe it's a better approach.